hudi-common/src/main/java/org/apache/hudi/common/model/PartialUpdateAvroPayload.java

```diff
@@ -93,10 +93,6 @@ public PartialUpdateAvroPayload(GenericRecord record, Comparable orderingVal) {
     super(record, orderingVal);
   }
 
-  public PartialUpdateAvroPayload(Option<GenericRecord> record) {
-    super(record); // natural order
-  }
-
```

Comment:
So why the natural order combining is removed? Can you elaborate a little more the use case?